lib/asciidoctor/latex/environment_block.rb

```diff
@@ -153,7 +152,7 @@ def process parent, reader, attrs
         warn "for rode = code, attrs = #{attrs}".cyan
         block = create_block parent, :listing, reader.lines, attrs
       else
-        block = create_block parent, :environment, reader.lines, attrs
+        block = create_block parent, :environment, reader.lines, attrs, content_model: :compound
```

Comment:
This is correct. The default content model is `:simple`. By making it `:compound`, you allow for nested blocks.